packages/Confirmation/src/components/TriggerButton/TriggerButton.js

```diff
@@ -6,15 +6,17 @@ const propTypes = {
   children: PropTypes.node,
   confirmId: PropTypes.string,
   icon: PropTypes.node,
```

Comment:
Why is `icon` a separate prop? Cant it just be a child?

Reply:
@mikrotron I recall we discussed this a while ago and settled on it together, do you recall a specific reason?

Reply:
I think for the `<DropdownMenu>` we ended up moving away from having an `icon` prop... it's not a prop anymore for `<DropdownMenu.Trigger>`:
https://github.com/acl-services/paprika/blob/master/packages/DropdownMenu/src/components/Trigger/Trigger.js

But we DID need a `buttonType` prop, which accepted `"icon"` as a possible value.

Reply:
So should we follow the same pattern here? I'm indifferent.

Reply:
Yeah i've now updated the confirmation components `TriggerButton` component to use the same api as the dropdownMenus `trigger` component. In a future PR I shall consolidate so that both DropdownMenu and the Confirmation component use the same code for `Trigger` Component.